core/components/atoms/avatarGroup/AvatarPopperBody.tsx

```diff
@@ -10,13 +10,13 @@ const AvatarPopperBody = (props: any) => {
   }
 
   return (
-    <div className="py-6 pr-4 pl-6">
+    <div className="px-4 py-3">
```

Comment:
According to design padding should be of `16px`. Please verify it once 
<img width="491" alt="image" src="https://github.com/innovaccer/design-system/assets/46045493/724a8162-1b6d-4b71-89bd-500814b5b2c4">